dotcom-rendering/cypress.json

```diff
@@ -1,9 +1,21 @@
 {
     "viewportWidth": 1500,
     "viewportHeight": 860,
-    "baseUrl": "http://localhost:3030/",
+    "baseUrl": "http://localhost:9000/",
     "video": false,
     "chromeWebSecurity": false,
-    "blockHosts": "*ophan.theguardian.com",
+    "blockHosts": [
+		"*ophan.theguardian.com",
+		"pixel.adsafeprotected.com",
+		"*permutive.com",
+		"*adnxs.com",
+		"*adsystem.com",
+		"*casalemedia.com",
+		"*pubmatic.com",
+		"*360yield.com",
+		"*omnitagjs.com",
+		"*the-ozone-project.com",
+		"*openx.net"
+	],
```

Comment:
I added these when debugging but maybe there's benefit to keeping them?

Reply:
I think it makes sense if it could contribute to removing the variability of some cypress tests!